docker/reverseproxy/conf.d/nginx.conf

```diff
@@ -0,0 +1,69 @@
+server {
+    listen 443 ssl;
+
+    server_name monitoring.localzero.net;
+
+    client_max_body_size 100m;
+
+    location / {
+        proxy_pass http://nginx-production:8080;
+        proxy_set_header Host $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    }
+
+    include /etc/nginx/conf.d/ssl.conf;
+}
+
+server {
+    listen 80;
+
+    server_name monitoring.localzero.net;
+
+    location / {
+        return 301 https://$host$request_uri;
+    }
+}
+
+server {
+    listen 443 ssl;
+
+    server_name monitoring-test.localzero.net;
+
+    client_max_body_size 100m;
+
+    location / {
+        proxy_pass http://nginx-testing:8080;
+        proxy_set_header Host $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    }
+
+    include /etc/nginx/conf.d/ssl.conf;
+}
+
+server {
```

Comment:
Can this be configured together with monitoring.localzero.net?

Reply:
... and also with the acme.sh proxy?

Reply:
I don't know if it knows then in the redirect which host to use :thinking:

Reply:
You already did the same thing with localhost and -test: just let the server respond to all host names and redirect to the right hosts and ports depending on the path, and it should work. I'll adjust it that way.